nixnet/types.py

```diff
@@ -153,6 +154,24 @@ class LinComm(LinComm_):
     pass
 
 
+PduProperties_ = collections.namedtuple(
+    'PDU_',
```

Comment:
Seems like the typename of the named tuple should be something like PDU_PROPERTIES_ instead of PDU_. Not sure if this naming really matters.

We use handle/ref to talk about an integer that the driver uses for the object. We are dealing with an actual Pdu object here; not a ref. Change to:

```
['pdu', 'start_bit', 'update_bit'])
```